### ISSUE TITLE
a simple tutorial for common redirecting examples?

### ISSUE THREAD

**Issue by abubelinha** [perma-id/w3id.org#1530]
I would appreciate a simple guide, or a list of .htaccess files, showing examples of how to do some quite common redirections.

w3id.org/myname  --->   my-current.domain/my-site-path/my-file.php
w3id.org/myname?a=1&b=4  ---> my-current.domain/my-site-path/my-file.php?a=1&b=4
w3id.org/myname/a_1/b_4  ---> my-current.domain/my-site-path/my-file.php?a=1&b=4
w3id.org/myname/site1  ---> my-current.domain/my-site-path/site1
w3id.org/myname/anothersite  ---> my-current.domain/my-site-path/anothersite

... and so on

Perhaps this is such a basic question, but there are no links in w3id.org to any good tutorial for beginners.

**Comment by davidlehn** [comment]
Yes, this would be good.  It's been on my todo list since forever.  Lots of things have been copied around that are probably not optimal.  I think the examples should be live working ones and in the an "example" directory.

**Comment by abubelinha** [comment]
Thanks @davidlehn 

Some other related thoughts:

- Nothing is said about "what is an .htaccess file"

I understand it stands for an Apache web server special file, used to handle redirections from urls named after the folder where the .htaccess file is placed. 

But I might be wrong: nothing is said about Apache, anywhere.  

- Should we look at any particular Apache web server version?
I don't know if the directives used inside .htaccess files could be different depending on the web server version.

**Comment by bact** [comment]
- [x] w3id.org/myname ---> my-current.domain/my-site-path/my-file.php [Covered]
- [x] w3id.org/myname?a=1&b=4 ---> my-current.domain/my-site-path/my-file.php?a=1&b=4 [Covered; not by individual parameters, but the entire query string]
- [ ] w3id.org/myname/a_1/b_4 ---> my-current.domain/my-site-path/my-file.php?a=1&b=4
- [x] w3id.org/myname/site1 ---> my-current.domain/my-site-path/site1 [Covered]
- [x] w3id.org/myname/anothersite ---> my-current.domain/my-site-path/anothersite [Covered]
